web/src/rest/publish.ts

```diff
@@ -128,24 +128,24 @@ const publishRest = new Vue({
       const version = (await this.versions(identifier, { page: count, page_size: 1 })).results[0];
       return girderize(version);
     },
-    async dandisets(params) {
+    async dandisets(params: any = undefined) {
       return client.get('dandisets/', { params });
     },
-    async createDandiset(name, description) {
+    async createDandiset(name: string, description: string) {
       const metadata = { name, description };
       return client.post('dandisets/', { name, metadata });
     },
-    async owners(identifier) {
+    async owners(identifier: string) {
       return client.get(`dandisets/${identifier}/users/`);
     },
-    async setOwners(identifier, owners) {
+    async setOwners(identifier: string, owners: [any]) {
```

Comment:
Is `owners` supposed to be an array? In that case, the proper syntax is:

```suggestion
    async setOwners(identifier: string, owners: any[]) {
```

What you've done is type `owners` as a [tuple](https://www.typescriptlang.org/docs/handbook/basic-types.html#tuple) of length 1, which I'm guessing is not what you intended. Also, if we add the `User` type as I mentioned above, this could become:
```suggestion
    async setOwners(identifier: string, owners: User[]) {
```

Unless an `Owner` here doesn't conform to the same spec as `User`.